app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee.erb

```diff
@@ -100,6 +101,19 @@ ShortFormApplicationService = ($localStorage) ->
   Service.cancelHouseholdMember = ->
     householdMembers = Service.householdMembers.filter (m) ->
       (m != Service.householdMember && m.id != Service.householdMember.id)
+    # clear out preferences where this householdMember was selected
```

Comment:
@kaptron coolio. Yeah I am gonna merge this for sure, thanks so much for the heads up!